src/figuresClasses.ts

```diff
@@ -1,19 +1,101 @@
+type Shapes = "triangle" | "circle" | "rectangle";
+type Colors = "red" | "green" | "blue";
+type colorOrNumber = Colors | number;
+
 export interface Figure {
+  shape: Shapes;
+  color: Colors;
+  getArea(): number;
+}
+
+class Controller {
+  static lengthIsLessThanZero = (...params: colorOrNumber[]): boolean => {
```

Comment:
you can move these methods from `Controller` to `Figure` class and use them other classes because you're implementing the base `Figure` class
```suggestion
  shape: Shapes;
  color: Colors;
  getArea(): number;
  
  static lengthIsLessThanZero = (...params: colorOrNumber[]): boolean => {
```